src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -4273,60 +4273,60 @@ module Queries = struct
       ~resolve:(fun { ctx = mina; _ } () state_hash_opt block_height_opt ->
         let open Deferred.Result.Let_syntax in
         let runtime_config = Mina_lib.runtime_config mina in
-        let%bind breadcrumb =
+        let%bind best_tip =
```

Comment:
This change just extracts `best_tip` retrieval to the top of function's definition